Framework/Algorithms/src/DiffractionFocussing2.cpp

```diff
@@ -581,7 +581,7 @@ void DiffractionFocussing2::determineRebinParameters() {
       throw std::runtime_error(mess.str());
     }
     // This log step size will give the right # of points
-    step = (log(Xmax) - log(Xmin)) / nPoints;
+    step = expm1((log(Xmax) - log(Xmin)) / nPoints);
```

Comment:
I confirm that this is the correct step - the previous code was valid only for small `(log(xmax)-log(xmin))/N`.
For some ENGIN-X data this made a difference in the ratio between points of ~2.5E-6 % - but obviously this was enough to break quite a few tests!